src/Bicep.Core.UnitTests/Bicep.Core.UnitTests.csproj

```diff
@@ -2,21 +2,16 @@
 
   <PropertyGroup>
     <IsPackable>false</IsPackable>
+    <OutputType>Exe</OutputType>
   </PropertyGroup>
```

Comment:
I got a test failure when running this test suite locally on your branch - not sure how this test was ever working. I've pushed a fix to your branch.

Reply:
The regression is fixed.